tests/conditional_compilation/tests_utils.py

```diff
@@ -23,3 +24,17 @@ def write_session_info(path: Path = Path(getsourcefile(lambda: 0)).parent / SESS
                        data: dict = None):
     with open(path, "w") as json_file:
         json.dump(data, json_file, indent=4)
+
+
+def run_infer(model, out_file, install_dir):
+    """ Function running inference
+    """
+
+    returncode, output = cmd_exec(
+        [sys.executable,
+         str((Path(getsourcefile(lambda: 0)) / ".." / "tools" / "infer_tool.py").resolve()),
```

Comment:
We calculating the infer_tool path twice. Should we make a fixture to remove such duplication?